src/login/logind.c

```diff
@@ -299,11 +300,13 @@ static int manager_enumerate_linger_users(Manager *m) {
 
         FOREACH_DIRENT(de, d, return -errno) {
                 int k;
+                _cleanup_free_ char *n = NULL;
 
                 if (!dirent_is_file(de))
                         continue;
-
-                k = manager_add_user_by_name(m, de->d_name, NULL);
+                k = cunescape(de->d_name, 0, &n);
```

Comment:
please just do:

```c
k = cunescape(de->d_name, 0, &n);
if (k < 0) {
        log_warning_errno(k, "Failed to unescape username '%s', ignoring.", de->d_name);
```

which is how we usually do that these days.

Reply:
Thanks. Force-pushed a correction.